### PR TITLE
[enh] Improve security docs

### PR DESCRIPTION
**Pull request by nqb**

### CODE REVIEW

**Comment by nqb** [comment]
For the 3.3.3 version, it would be nice to update ssh.md according to https://github.com/YunoHost/yunohost/pull/518.

**Comment by alexAubin** [comment]
@nqb : it's not yet released, this will be in 3.4.x

File: security.md [hunk 1]
```diff
@@ -147,10 +147,11 @@ systemctl restart ssh
 ---
 
 ### Disable YunoHost API
-YunoHost administration is accessible through an **HTTP API**, served on the 6787 port by default. It can be used to administrate a lot of things on your server, so malicious actors can also use it to damage your server. The best thing to do, if you know how to use the [command-line interface](/commandline), is to deactivate the `yunohost-api` service.
+YunoHost administration is accessible through an **HTTP API**, served on the 6787 port by default (only on `localhost`). It can be used to administrate a lot of things on your server, so malicious actors can also use it to damage your server. The best thing to do, if you know how to use the [command-line interface](/commandline), is to deactivate the `yunohost-api` service.
```
Comment (nqb) [inline security.md]: In that case, you will lose access to YunoHost admin interface.

**Comment by nqb** [comment]
> @nqb : it's not yet released, this will be in 3.4.x

Sorry, I didn't see the milestone.

File: security_fr.md [hunk 2]
```diff
@@ -162,10 +162,11 @@ systemctl restart ssh
 
 ### Désactivation de l’API YunoHost
 
-YunoHost est administrable via une **API HTTP**, servie sur le port 6787 par défaut. Elle permet d’administrer une grande partie de votre serveur, et peut donc être utilisée à des **fins malveillantes**. La meilleure chose à faire si vous êtes habitués aux lignes de commande est de désactiver le service `yunohost-api`, et **utiliser la [ligne de commande](/commandline_fr)** en SSH.
+YunoHost est administrable via une **API HTTP**, servie sur le port 6787 par défaut (seulement sur `localhost`). Elle permet d’administrer une grande partie de votre serveur, et peut donc être utilisée à des **fins malveillantes**. La meilleure chose à faire si vous êtes habitués aux lignes de commande est de désactiver le service `yunohost-api`, et **utiliser la [ligne de commande](/commandline_fr)** en SSH.
 
 ```bash
-sudo service yunohost-api stop
+sudo systemctl disable yunohost-api
```
Comment (nqb) [inline security_fr.md]:
```suggestion
sudo systemctl mask yunohost-api
```
`disable` action doesn't prevent to start service by hand.

File: security.md (hunk 1 above)
Comment (zamentur) [inline security.md]: "YunoHost administration is accessible through an **HTTP API**, served on the URL /yunohost/api/ ." ?

File: security_fr.md (hunk 2 above)
Comment (zamentur) [reply security_fr.md]: In some case yunohost restart yunohost-api, if we mask it, is there a risk that the command fails ?

Comment (nqb) [reply security_fr.md]: Using `mask` will prevent all restart of this service.

**Comment by nqb** [comment]
Ready for review.

File: security.md (hunk 1 above)
Comment (nqb) [reply security.md]: Is it the right place to mention this ?

**Pull request merged**